integration_tests/e2e/conditions/create/createConditions.cy.ts

```diff
@@ -30,15 +30,95 @@ describe('Create Conditions', () => {
+      .submitPageTo(SelectConditionsPage)
+      .hasErrorCount(2)
+      .hasFieldInError('VISUAL_IMPAIR_conditionNames')
+      .hasFieldInError('LONG_TERM_OTHER_conditionNames')
+      // specify the conditions in the conditional fields
+      .specifyCondition(ConditionType.VISUAL_IMPAIR, 'Colour blindness')
+      .specifyCondition(ConditionType.LONG_TERM_OTHER, 'Arthritis')
+      .submitPageTo(ConditionsDetailsPage)
+
+    Page.verifyOnPage(ConditionsDetailsPage)
+      // submit the page without answering the questions to trigger a validation error
+      .submitPageTo(ConditionsDetailsPage)
+      .hasErrorCount(3)
+      .hasFieldInError('ADHD_details')
+      .hasFieldInError('VISUAL_IMPAIR_details')
+      .hasFieldInError('LONG_TERM_OTHER_details')
+      // enter the condition details
+      .enterConditionDetails(
+        ConditionType.ADHD,
+        'Has trouble concentrating and sitting still for long periods. Easily distracted.',
+      )
+      .enterConditionDetails(ConditionType.VISUAL_IMPAIR, 'Has red-green colour blindness.')
+      .enterConditionDetails(ConditionType.LONG_TERM_OTHER, 'Acute arthritis in the right arm and hand. Cause unknown.')
+      .submitPageTo(ConditionsPage)
+
+    // Then
+    Page.verifyOnPage(ConditionsPage).hasSuccessMessage('Condition(s) updated')
+
+    cy.wiremockVerify(
+      postRequestedFor(urlEqualTo(`/support-additional-needs-api/profile/${prisonNumber}/conditions`)) //
+        .withRequestBody(
+          matchingJsonPath(
+            '$[?(' +
+              '@.conditions.size() == 3 && ' +
+              "@.conditions[0].prisonId == 'BXI' && " +
+              "@.conditions[0].conditionTypeCode == 'ADHD' && " +
+              "@.conditions[0].source == 'SELF_DECLARED' && " +
+              '!@.conditions[0].conditionName && ' +
+              "@.conditions[0].conditionDetails == 'Has trouble concentrating and sitting still for long periods. Easily distracted.' && " +
+              "@.conditions[1].prisonId == 'BXI' && " +
+              "@.conditions[1].conditionTypeCode == 'VISUAL_IMPAIR' && " +
+              "@.conditions[1].source == 'SELF_DECLARED' && " +
+              "@.conditions[1].conditionName == 'Colour blindness' && " +
+              "@.conditions[1].conditionDetails == 'Has red-green colour blindness.' && " +
+              "@.conditions[2].prisonId == 'BXI' && " +
+              "@.conditions[2].conditionTypeCode == 'LONG_TERM_OTHER' && " +
+              "@.conditions[2].source == 'SELF_DECLARED' && " +
+              "@.conditions[2].conditionName == 'Arthritis' && " +
+              "@.conditions[2].conditionDetails == 'Acute arthritis in the right arm and hand. Cause unknown.' " +
+              ')]',
+          ),
+        ),
+    )
+  })
+
+  it('should create a prisoners Conditions, triggering validation on every screen given user has permission to create diagnosed conditions', () => {
```

Comment:
_This_ is a new test; though is a copy and paste of the above one (which is probably why the diff is slightly confused)
This test is for users who _do_ have the permission to create diagnosed conditions, so the signIn stub is slightly different, as is the page heading assertion (L121)

The rest of this test is identical, but will change when we do RR-1706 because that adds some extra questions to the Details page, and this test will need to cater for that. It will also mean the answers to those questions are sent to the API, and that will need to be asserted in this test.
But thats all going to come in RR-1706